### DISCUSSION TITLE
Could not find module 'NYCGEO.dll'

### DISCUSSION THREAD

**Discussion by omarvelous89** [ishiland/python-geosupport#34] [Q&A] [answered]
Hi everyone,

Some background. I am using a Miniconda installation on 64-bit Windows machine and python 3.

After installing Geosupport Desktop Edition (Path: C:\Program Files), I opened a Jupyter Notebook and attempted to run the first piece of code to test the installation. Code was taken from documentation page https://python-geosupport.readthedocs.io/en/latest/usage.html

```python
# Import the library and create a `Geosupport` object.
from geosupport import Geosupport
g = Geosupport()

# Call the address processing function by name
result = g.address(house_number=125, street_name='Worth St', borough_code='Mn')
```

This resulted in an error [**GeosupportError:** Could not find module 'NYCGEO.dll' (or one of its dependencies). Try using the full path with constructor syntax. You are currently using a 64-bit Python interpreter. Is the installed version of Geosupport 64-bit?

I confirmed my installation of Geosupport is 64 bit and my machine is 64 bit so I then attempted to define the path explicitly
```python
# Import the library and create a `Geosupport` object.
from geosupport import Geosupport
g = Geosupport(geosupport_path="C:\Program Files\Geosupport Desktop Edition")

# Call the address processing function by name
result = g.address(house_number=125, street_name='Worth St', borough_code='Mn')
```
which resulted in the same error. I tried other path pointing directly to the DLL file but with no success. Not sure how to proceed

**Comment by ishiland** [comment, marked as answer]
This is probably related to #30. I just pushed an update to the pypi repository, if you could try to re-install the latest version (1.0.9) and see if that helps.

**Reply by omarvelous89** [reply]
Thank you so much Ian! That did the trick. I appreciate the quick reply :)
